SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestCharacterMovement/SpatialTestCharacterMovement.cpp

```diff
@@ -117,6 +92,10 @@ void ASpatialTestCharacterMovement::PrepareTest()
 
 			PlayerCharacter->AddMovementInput(FVector(1, 0, 0), 1.0f);
 
+			bCharacterReachedDestination =
```

Comment:
I find this test a bit unintuitive to read. Could you replace it with something based on FPlane::PlaneDot ?

Reply:
For me this is the simplest test to check if a character has reached a particular destination: `FVector::Dist(Destination, PlayerCharacter->GetActorLocation()) < 20.0f;`

Reply:
Added the FPlane as suggested to make the test easier to understand.